File: examples/node/src/07_get_order.ts
import { createClient, getEnv } from "./_mcp_sse_client.js";

async function main() {
  const client = createClient();
  const orderId = getEnv("DEMO_ORDER_ID", "<ORDER_ID>");
  const email = getEnv("DEMO_EMAIL", "user@example.com");

  const res = await client.callTool("getOrder", { orderId, email });
  console.log(JSON.stringify(res, null, 2));

  client.close();
}

main().catch((e) => {
  console.error(e);
  process.exit(1);
});

File: examples/python/src/07_get_order.py
import json
from client import McpSseClient, env

def main():
    order_id = env("DEMO_ORDER_ID", "<ORDER_ID>")
    email = env("DEMO_EMAIL", "user@example.com")

    c = McpSseClient(env("MCP_BASE_URL"))
    res = c.call_tool("getOrder", {"orderId": order_id, "email": email})
    print(json.dumps(res, indent=2))

if __name__ == "__main__":
    main()
